restful-ws/3.0/_index.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: "Jakarta RESTful Web Services 3.0 (under development)"
-date: 2020-01-15
+title: "Jakarta RESTful Web Services 3.0 (milestone)"
+date: 2020-05-26
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta RESTful Web Services provides a foundational API to develop web services
 following the Representational State Transfer (REST) architectural pattern.
 
 * [Jakarta RESTful Web Services 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.jaxrs/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta RESTful Web Services 3.0 Specification Document]() (PDF)
-* [Jakarta RESTful Web Services 3.0 Specification Document]() (HTML)
+* [Jakarta RESTful Web Services 3.0 Specification Document](./restful-ws-spec-3.0.pdf) (PDF)
```

Comment:
I would name your Milestone or RC spec deliverable with an identifier to indicate it's an early draft.  For example, restful-ws-spec-3.0-M1.pdf (or RCx, or draft, or something to indicate it's not the final version of the 3.0 spec).

Reply:
Thanks @kwsutter - I renamed the file to include "-M1".  The file themselves also mention that they are drafts - and also contain the "-M1" qualifier.